packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.jsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const TurbosInfo = ({ className, currency, has_stop_loss, max_stake = 100, min_stake = 1 }) => {
```

Comment:
Why do you define the variables in the store and give them a default value in the props?
Keep them in one place only to have only a single source or truth.

Reply:
Because currently we do not get these values from BE, so I hardcoded them in store and also add default value in component just to be sure. When we able to get those values from BE, I'll remove all hard coded parts, but it a good idea to clean up default props values right now

Reply:
Ok then leave a todo comment to update it once you get the values from BE